integration/index.test.js

```diff
@@ -0,0 +1,79 @@
+const childProcess = require('child_process');
+const fetch = require('node-fetch');
+
+let samProcess;
+const port = 5000;
```

Comment:
I know this port is used only for local development, but does it make sense to come from an environment variable, from `.env` file or something?